tests/test_workers/test_tasks/test_build.py

```diff
@@ -254,12 +255,30 @@ def test_push_image(mock_si, mock_run_cmd, mock_glps, schema_version):
     assert destination in push_args
 
     if schema_version == 1:
-        skopeo_args = mock_run_cmd.mock_calls[1][1][0]
-        assert skopeo_args == ['skopeo', 'copy', '--format', 'v2s2', destination, destination]
+        mock_fsv.assert_called_once_with(destination)
     else:
         mock_run_cmd.assert_called_once()
 
 
+@mock.patch('iib.workers.tasks.build.run_cmd')
+def test_fix_schema_version_fail_retry_success(mock_run_cmd):
+    destination = 'some_destination'
+    build._fix_schema_version(destination)
+    skopeo_args = mock_run_cmd.mock_calls[0][1][0]
+    assert skopeo_args == ['skopeo', 'copy', '--format', 'v2s2', destination, destination]
+    mock_run_cmd.assert_called_once()
+
+
+@mock.patch('iib.workers.tasks.build.run_cmd')
+def test_fix_schema_version_fail_max_retries(mock_run_cmd):
```

Comment:
Optional: This test can be removed since you have `test_retry`.

Reply:
yeah, but we would anyways want a failure test. So I thought might as well try the retry as well. Let's keep it.